### PR TITLE
Add load() test

### PR DESCRIPTION
**Pull request by higashi000**
issues: #75 

add load function test.
It is expected test will faile.
Because, it is not exist Vim script file in main branch.

### CODE REVIEW

File: denops_std/helper/load_test.ts [hunk 1]
```diff
@@ -17,10 +17,54 @@ test({
...
+    }
+    await assertThrowsAsync(
+      async () => {
+        await load(
+          denops,
+          new URL("./load_test_not_exists.vim", import.meta.url),
+        );
+      },
+      undefined,
+      `Failed to call 'call' with ["denops#api#cmd","call execute(l:__denops_internal_command, '')",{"scriptPath":"${filePath}","__denops_internal_command":["execute printf('source %s', fnameescape(scriptPath))"]}]: Error: Failed to call 'nvim_call_function' with ["denops#api#cmd",["call execute(l:__denops_internal_command, '')",{"scriptPath":"${filePath}","__denops_internal_command":["execute printf('source %s', fnameescape(scriptPath))"]}]]: [0,"Vim(source):E484: Can't open file ${filePath}`,
```
Comment (lambdalisue) [inline denops_std/helper/load_test.ts:47]: You don't need to specify full sentence here.

```
        `Failed to call 'call' with`,
```

seems enough.

File: denops_std/helper/load_test.ts [hunk 2]
```diff
@@ -17,10 +17,54 @@ test({
...
+        await load(
+          denops,
+          new URL(
+            "https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test_not_exists.vim",
+            import.meta.url,
+          ),
+        );
+      },
+      Error,
+      "Failed to fetch 'https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test_not_exists.vim'",
```
Comment (lambdalisue) [inline denops_std/helper/load_test.ts:67]: This one as well.

```
    "Failed to fetch"
```

seems enough.

File: denops_std/helper/load_test.ts [hunk 3]
```diff
@@ -17,10 +17,54 @@ test({
     await load(
       denops,
       new URL(
-        "https://raw.githubusercontent.com/vim-denops/deno-denops-std/main/denops_std/helper/load_test.vim",
+        "https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test.vim",
```
Comment (lambdalisue) [inline denops_std/helper/load_test.ts:20]: Please define a constant variable for this URL.

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: denops_std/helper/load_test.ts (hunk 3 above)
Comment (higashi000) [reply denops_std/helper/load_test.ts]: Fixed it with c1fc317.

File: denops_std/helper/load_test.ts (hunk 2 above)
Comment (higashi000) [reply denops_std/helper/load_test.ts]: Fixed it with c1fc317.

File: denops_std/helper/load_test.ts (hunk 1 above)
Comment (higashi000) [reply denops_std/helper/load_test.ts]: Fixed it with c1fc317.

File: denops_std/helper/load_test.ts [hunk 4]
```diff
@@ -14,13 +14,54 @@ test({
   mode: "any",
   name: "load() load remote Vim script file",
   fn: async (denops) => {
+    const loadScriptUrl =
+      "https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test.vim";
```
Comment (lambdalisue) [inline denops_std/helper/load_test.ts:18]: Please define the base URL in file-level and use it like

```typescript
const loadScriptUrl = new URL("load_test.vim", loadScriptUrlBase);
await load(denops, loadScriptUrl);
// ...
const loadScriptUrl = new URL("load_test_not_exists.vim", loadScriptUrlBase);
await load(denops, loadScriptUrl);
```

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: denops_std/helper/load_test.ts (hunk 4 above)
Comment (higashi000) [reply denops_std/helper/load_test.ts]: Fixed it with 6d7fdec.

**Review by lambdalisue — APPROVED** [review]
LGTM

**Pull request merged**